edb/server/pgcon/pgcon.pyx

```diff
@@ -103,6 +103,7 @@ cdef dict POSTGRES_SHUTDOWN_ERR_CODES = {
 }
 
 cdef bytes INIT_CON_SCRIPT = None
+cdef object EMPTY_SQL_STATE = json.dumps({}).encode('utf-8')
```

Comment:
```suggestion
cdef object EMPTY_SQL_STATE = "{}".encode('utf-8')
```

? I admire the readability of this but it does make the startup slower, however minuscule that would be